CHANGELOG.md

```diff
@@ -4,6 +4,182 @@ All notable changes to Router will be documented in this file.
+
+This adds a `uuid_v4()` and `unix_now()` function to obtain a UUID and Unix timestamp, respectively.
+
+By [@lleadbet](https://github.com/lleadbet) in https://github.com/apollographql/router/pull/2617
+
+### Show option to "Include Cookies" in Sandbox
+
+This adds default support for using the "Include Cookies" toggle when working with Embedded Sandbox.
+
+By [@esilverm](https://github.com/esilverm) in https://github.com/apollographql/router/pull/2553
+
+### Add a metric to track the cache size ([Issue #2522](https://github.com/apollographql/router/issues/2522))
+
+We've introduced a new `apollo_router_cache_size` metric that reports the current size of in-memory caches.  Like [other metrics](https://www.apollographql.com/docs/router/configuration/metrics), it is available via OpenTelemetry Metrics including Prometheus scraping.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2607
+
+### Add a rhai global variable resolver and populate it ([Issue #2628](https://github.com/apollographql/router/issues/2628))
+
+Rhai doesn't place _constants_ into the global scope, instead requiring that functions be pure. We only became aware of this limitation this week, mainly as a consequence of people having difficulties with the experimental `APOLLO_AUTHENTICATION_JWT_CLAIMS` constant.
+
+Our fix is to introduce a new global [variable resolver](https://rhai.rs/book/engine/var.html) and populate it with a `Router` global constant. It currently has three members:
+
+ - `APOLLO_START -> should be used in place of `apollo_start`
+ - `APOLLO_SDL -> should be used in place of `apollo_sdl`
+ - `APOLLO_AUTHENTICATION_JWT_CLAIMS`
+
+You access a member of this variable as follows:
+
+```
+   let my_var = Router.APOLLO_SDL;
+```
+
+We are removing the _experimental_ `APOLLO_AUTHENTICATION_JWT_CLAIMS` constant, but we will **retain the existing non-experimental constants** for purposes of backwards compatibility.
+
+
+We recommend that you shift to the new global constants since we will remove them in a major breaking change release in the future.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/2627
+
+### Activate TLS for Redis cluster connections ([Issue #2332](https://github.com/apollographql/router/issues/2332))
+
+This adds support for TLS connections in Redis Cluster mode, by applying it when the URLs use the `rediss` schema.
+
+By [@Geaal](https://github.com/Geal) in https://github.com/apollographql/router/pull/2605
+
+### Make `terminationGracePeriodSeconds` property configurable in the Helm chart
+
+The `terminationGracePeriodSeconds` property is now configurable on the `Deployment` object in the Helm chart.
+
+This can be useful when adjusting the default timeout values for the router, and should always be a value slightly bigger than the timeout in order to ensure no requests are closed prematurely on shutdown.
```

Comment:
```suggestion
This can be useful when adjusting the default timeout values for the Router, and should always be a value slightly bigger than the Router timeout in order to ensure no requests are closed prematurely on shutdown.

The Router timeout is configured via `traffic_shaping`

```yaml
traffic_shaping: 
  router: 
    timeout: ...
```
```